tasks/build.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+# Elastic Filebeat is might be extracted to it's own role later on.
+# For MVP we will install it heres
+- name: Add Elastic APT key
+  become: yes
+  apt_key:
+    keyserver: hkp://keyserver.ubuntu.com:80
+    id: "D88E42B4"
+    state: present
+
+- name: Install Elastic repo
+  become: yes
+  apt_repository:
+    repo: "deb https://artifacts.elastic.co/packages/6.x/apt stable main"
+    state: present
+    update_cache: yes
+
+- name: Install Filebeat
+  become: yes
+  apt:
+    name: filebeat
```

Comment:
We should be able to specify a specific version here, not just default to latest